requirements.txt

```diff
@@ -3,5 +3,9 @@ Flask==0.12.2
 itsdangerous==0.24
 Jinja2==2.10
 MarkupSafe==1.0
+pluggy==0.6.0
+py==1.5.2
```

Comment:
Don't we also need `pytest`?

Reply:
Look in the `deps` for `[testenv:test]`. That's the great thing about Tox; you can define envs super easily for each step or even run the same step against a bunch of envs. E.G "How would my unit tests act in python 2.7 with these dependency versions?"

Reply:
So the idea is not to run `pytest` directly but to always call `tox` and let it install `pytest`?

Reply:
Yup! Anything you need to do on a regular basis you add to your `tox.ini`. You can: lint, unit test, build a docker image, anything. It will set up your venv and everything. You could even have a step that runs the server and never have to worry about your venv. :)

Reply:
Got it!